### ISSUE TITLE
"Create a sandbox for wine folders" seemingly ignored and forcing sandbox directory creation

### ISSUE THREAD

**Issue by MetalSkotty** [lutris/lutris#2741]
**Describe the bug**

The soft links to my home directory to save my game files are always replaced by Wine sandbox folders when I start a Wine and Wine-Steam game.

**Expected behavior**

The "Create a sandbox for wine folders" setting is turned off. The Wine application must refer to my home directory to store game saves.

**Current behavior**

With turned off "Create a sandbox for wine folders", starting a game via Wine/Wine-Steam does always forcefully replace any soft links to my home/documents directory.

**Steps to reproduce**

- Open the "Runner Options" for any Wine/Wine Steam game in Lutris 0.5.5
- Make sure "Create a sandbox for wine folders" is turned off for the game
- Go to e.g. `/home/*user*/.local/share/lutris/runners/winesteam/prefix64/drive_c/users/*user*/` and make sure your "My Documents" is actually a soft link to `/home/*user*/Documents`
- Start the game via Lutris
- See that the soft link was replaced by a normal directory in `/home/*user*/.local/share/lutris/runners/winesteam/prefix64/drive_c/users/*user*/`

**System information**

Fedora 31, Kernel 5.15, German locale. Details see file below.
[lutris-issue-report.zip](https://github.com/lutris/lutris/files/4467247/lutris-issue-report.zip)

**Comment by tannisroot** [comment]
Can you rephrase your reproduction steps and possibly record a video to showcase the issue? What you described just doesn't make any sense and I can't reproduce it.

**Comment by MetalSkotty** [comment]
I rephrased it. Hopefully this is more clear now.

**Comment by tannisroot** [comment]
Can you check if this is fixed in master?

**Comment by tannisroot** [comment]
Fixed in master apparently

**Comment by MetalSkotty** [comment]
Confirming the problem was fixed.
